### PR TITLE
Task 5. Create CRUD<T> interface

### PR DESCRIPTION
**Pull request by AmorBielyi**

### CODE REVIEW

File: src/main/java/com/cursor/dao/interfaces/CRUD.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package com.cursor.dao.interfaces;
+
+import java.util.List;
+
+public interface CRUD<T> {
+    void create(T entity);
```
Comment (Julia1910) [inline src/main/java/com/cursor/dao/interfaces/CRUD.java:6]: Isn't "create" must be boolean?

File: src/main/java/com/cursor/dao/interfaces/CRUD.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+    List<T> getAll();
+
+    T findById(int id);
+
+    boolean delete(int id);
+}
```
Comment (Julia1910) [inline src/main/java/com/cursor/dao/interfaces/CRUD.java:15]: Also must be method "edit"

File: src/main/java/com/cursor/dao/interfaces/CRUD.java (hunk 1 above)
Comment (AmorBielyi) [reply src/main/java/com/cursor/dao/interfaces/CRUD.java:6]: Yep, overlooked. Thx)

Comment (AmorBielyi) [reply src/main/java/com/cursor/dao/interfaces/CRUD.java]: Fixed by new commit

**Review by Julia1910 — APPROVED** [review]

**Review by Julia1910 — APPROVED** [review]

**Review by illia-nerdysoft — APPROVED** [review]

**Review by Barburon — APPROVED** [review]

**Review by ShvedovIllia — APPROVED** [review]

**Review by lazermann111 — APPROVED** [review]

**Pull request merged**